### ISSUE TITLE
Heatmap example is blank

### ISSUE THREAD

**Issue by mnesler** [heremaps/harp.gl#2365]
Had anyone had luck with a heatmap?

We noticed the documentation is missing
https://www.harp.gl/docs/master/doc/modules/harp_examples.geojsonheatmapexample.html

We are trying to draw cubes at this point.

Thanks,

Max

**Comment by nzjony** [comment]
@mnesler , are you referring to: https://www.harp.gl/docs/master/examples/#datasource_geojson_choropleth.html ?

**Comment by mnesler** [comment]
@nzjony thank you for the reply.  Is the heapmap link supposed to be the choropleth link?

**Comment by nzjony** [comment]
@mnesler , yes, I think heat map is the wrong term, it should be choropleth, see: https://en.wikipedia.org/wiki/Choropleth_map

**Comment by mnesler** [comment]
@nzjony heatmap is the term most mapping libs use.  Choropleth is neat, but you already have the geometry defined as borders of a state.

I have to change from Deck.gl which has a feature for 3d heatmaps; you just provide geopoints and the library draws a three dimensional 'grid'.  This feature is either hidden or missing in Here.  Our team is now considering developing this feature.   

Will this feature described above be in Heremaps / Harp anytime soon?
